time-duration-picker/src/main/java/mobi/upod/timedurationpicker/TimeDurationPicker.java

```diff
@@ -38,14 +46,18 @@
     private final Button[] numPadButtons;
     private final Button numPadMeasureButton;
     private OnDurationChangedListener changeListener = null;
+    private TextView secondsLabel;
+    private TextView hoursLabel;
+    private TextView minutesLabel;
 
     /**
      * Implement this interface and set it using #setOnDurationChangeListener to get informed about input changes.
      */
     public interface OnDurationChangedListener {
         /**
          * Called whenever the input (the displayed duration string) changes.
-         * @param view the view that fired the event
+         *
```

Comment:
I find this funny since I commented that I tried to not make unnecessary changes. Will roll these back.